frontend/src/routes/(app)/(third-party)/requirement-assessments/[id=uuid]/edit/+page.svelte

```
@@ -141,6 +144,40 @@
+			const applied_controls = await res.json();
+			const reference_controls_list = reference_controls.map((rc) => rc.id);
+			applied_controls_options = applied_controls.results.map((t: any) => {
+				const label = `${t.folder.str}/${t.name}`;
+				return {
+					value: t.id,
+					label,
+					translatedLabel: label,
+					suggested: reference_controls_list.includes(t.reference_control?.id),
+					ref: t.reference_control?.id
+				};
+			}).sort((a, b) => {
```

Biome (2.4.4)

[error] This callback passed to sort() iterable method should always return a value.

(lint/suspicious/useIterableCallbackReturn)